spec/requests/strategies_spec.rb

```diff
@@ -1,127 +1,128 @@
 # frozen_string_literal: true
 
-describe StrategiesController do
+describe 'Strategy', type: :request do
   let(:user) { create(:user) }
   let(:strategy) { create(:strategy, user: user) }
 
   describe '#index' do
     let(:strategy) { create(:strategy, name: 'test', user: user) }
 
     context 'when the user is logged in' do
-      include_context :logged_in_user
+      before { sign_in user}
 
       context 'when search params are provided' do
-        before { get :index, params: { search: 'test' } }
-
         it 'assigns @strategies' do
+          get strategies_path, params: { search: 'test' }
           expect(assigns(:strategies)).to eq([strategy])
```

Comment:
Hmm I guess it really is no problem, because there are other render/data expectations in this context, but I'm not 100% sold on `assigns` matcher for request specs 🤔 
I've always believed it's best to just test the input and output for each possible case. But I guess it isn't hurting anyone still being there 🤷 lol ... especially if we're not rendering json, I guess that's one valid way to check that the data was actually assigned...hmmmm TIL

Reply:
Yeah in this [article](https://medium.com/just-tech/rspec-controller-or-request-specs-d93ef563ef11) that @bennpham linked in the original issue, it says that `assigns` and `assert_template` is deprecated. Is there another way to check for the value of instance variables?

Reply:
As I mentioned to someone else on Slack, one way to check is check the actual response body (in the case of json) and confirm the returned items are what's expected.

```ruby
expect(JSON.parse(response.body['some_id']).to eq(expected_value)
```

For html expectations I'm a little more rusty, as most ofvmy experience in JSON apis

Reply:
I used the _strategy_ in that article in which I expect an attribute to be included in the body of the response.